model_builder.h

```diff
@@ -33,9 +33,12 @@ class ModelBuilder {
 
   WCollective spawnKeeper(WModel, AvatarInfo, bool regenerateMana, vector<string> introText);
 
-  static int getPigstyPopulationIncrease();
-  static int getStatuePopulationIncrease();
-  static int getThronePopulationIncrease();
+  static double getPigstyPopulationIncrease();
```

Comment:
All these constants feel like they belong as a part of the furniture data, rather than the model builder.

It might be fun to give every piece of furniture a small contribution to your overall maximum population. Instead of starting off with a maximum of 10, you could start off with a maximum of 1, and get up to 3 for beds, 3 for cages, 3 for grave stones. Then some of the other bits of furniture could give you 0.25 etc.

Reply:
Both are great ideas, I'll think about them.